tests/system/Models/DeleteModelTest.php

```diff
@@ -148,12 +148,12 @@ public function testOnlyDeleted(): void
     }
 
     /**
-     * If where condition is set, beyond the value was empty (0,'', NULL, etc.),
-     * Exception should not be thrown because condition was explicity set
+     * If where condition is set, even if the value is empty (0, NULL, etc.),
+     * Exception should not be thrown because condition is explicitly set.
```

Comment:
Maybe refactor this? Doesn't really matter, it is a comment on a test method, but it is difficult to read. Gherkin syntax:
```suggestion
     * Given an explicit empty value in the WHERE condition
     * When executing a soft delete
     * Then an exception should not be thrown
```